README.md

```diff
@@ -95,6 +95,94 @@ status:
   state: DEPLOYED
 ```
 
+## Minimum resource requirement
+
+The minimum memory and cpu requirements for each ixia-c components are captured in the following table. Kubernetes metrics server has been used to collect resource usage data. The memory represents the minimum working set memory required, and for protocol and traffic engines, varies depending on the number of colocated ports. The figures are in Mi or MB.
```

Comment:
Note sure why some words are showing up in red.

Reply:
Looks fine if you go through 
https://github.com/open-traffic-generator/ixia-c-operator/tree/dev-resource-reqm